src/cascade/executor/covariate_data.py

```diff
@@ -69,7 +69,29 @@ def add_country_covariate_to_observations_and_avgints(data, local_settings, epiv
     country_specs = {ccov for ccov in epiviz_covariates if ccov.study_country == "country"}
     for covariate_id in {evc.covariate_id for evc in country_specs}:
         ccov_ranges_df = data.country_covariates[covariate_id]
-        reference = reference_value_for_covariate_mean_all_values(ccov_ranges_df)
+
+        parent_ccov_ranges = ccov_ranges_df[ccov_ranges_df.location_id.isin([local_settings.parent_location_id])]
+        if not parent_ccov_ranges.empty:
+            CODELOG.warning ("****** FIXME -- The parent had ccov values -- population weight these *********")
```

Comment:
Can you remove this whitespace? Travis automatically checks for pep8 formatting, and it will say our build is failing if we merged this.

Reply:
You have to be kidding. But I'll take them out.